app/views/include/SubscriptionsGroup.scala.html

```diff
@@ -43,7 +43,7 @@
 <div class="govuk-accordion" data-module="govuk-accordion" id="@displayableAccordionId()">
   @apisSubscriptions.map { case apiSubscription =>
 
-    <div class="govuk-accordion__section @if(apiSubscription.hasSubscriptions) {govuk-accordion__section--expanded}" id="@displayableAccordionId()-section-@displayableSectionId(apiSubscription)">
+    <div class="govuk-accordion__section @if(apiSubscription.hasSubscriptions) {govuk-accordion__section--expanded}" id="@apiSubscription.apiServiceName-accordion-section">
```

Comment:
Genuine Question - does apiSubscription.apiServiceName work without {} i.e. without then trying to resolve the -xyz-abc stuff too.

Reply:
Yes, I'm literally just reverting the change to this element from my previous commit (https://github.com/hmrc/third-party-developer-frontend/commit/ed9f624bcea0e37616437c673f2368e10748cc01) and it definitely worked then and I've retested it....